### DISCUSSION TITLE
运行时出现错误

### DISCUSSION THREAD

**Discussion by shoucandanghehe** [HMBSbige/BilibiliLiveRecordDownLoader#93] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/51957264/121434770-645a5200-c9b0-11eb-8e85-f385c66d8ef3.png)

**Comment by shoucandanghehe** [comment]
点击视频处理栏时发生

**Reply by HMBSbige** [reply]
系统版本？

**Reply by shoucandanghehe** [reply]
win10专业版
21H1
19043.985

**Reply by HMBSbige** [reply]
确保是解压缩运行 `Win64` 版本的话，可能是目录没权限，或者被其他软件拦截

**Reply by shoucandanghehe** [reply]
用的是BilibiliLiveRecordDownLoader-3.7.1.7z

**Reply by shoucandanghehe** [reply]
没有其他杀毒软件，只有windows defender
目录权限应该也没啥问题，毕竟log都是正常读写的

**Reply by HMBSbige** [reply]
那你用 Win64 的版本吧，要么就重新安装最新的 .NET5.0 运行时

这种报错都是系统少东西

**Reply by shoucandanghehe** [reply]
我装下最新.net5.0吧
这报错也就出现这一次，应该是偶发问题

**Comment by HMBSbige** [comment, marked as answer]
这种报错应该是程序运行时 runtime 更新了，导致旧版本的 runtime 被卸载了😰

用 Self-Contained 的版本就不会出现这种问题
